### PR TITLE
@broskoski Move article to subapp

### PR DESCRIPTION
**Pull request by kanaabe**
Differentiates between `article` -- an individual article, and `articles` -- a collection of articles.

Also moves the email-related stuff to its own component. I'll have to work on getting the components to be less hard-coded and easier to reuse, but I think it's a good start

### CODE REVIEW

File: apps/article/stylesheets/index.styl [hunk 1]
```diff
@@ -57,4 +57,32 @@
...
-    margin auto
\ No newline at end of file
+    margin auto
+
+.articles-featured-articles
+  margin gutter-size auto
+  padding 20px 0
+  border-bottom 1px solid gray-lighter-color
+  h4
+    avant-garde()
```
Comment (broskoski) [inline apps/article/stylesheets/index.styl]: I would use the elan `avant-garde-size` mixins as much as possible! Not sure if it makes sense here (if there are some specific line-height requirements, etc) but those mixins are good for site-wide consistency.

File: assets/article.coffee [hunk 2]
```diff
@@ -0,0 +1,6 @@
+sd = require('sharify').data
+require('backbone').$ = $
+
+$ ->
+  if location.pathname.match '/article/.*'
```
Comment (broskoski) [inline assets/article.coffee]: Just curious, why is this necessary?

**Comment by broskoski** [comment]
Nice, looks like a good re-org and preview of things to come :)

None of the stuff above is blocking, feel free to address later.

File: assets/article.coffee (hunk 2 above)
Comment (kanaabe) [inline assets/article.coffee]: You're right, not necessary :)

File: apps/article/stylesheets/index.styl (hunk 1 above)
Comment (kanaabe) [inline apps/article/stylesheets/index.styl]: Ah yes, these styles I copied over from the `articles` subapp. Some older style choices remain.
